### PR TITLE
Fix for rolling_version_change where the version is an integer

### PR DESCRIPTION
**Pull request by sishbi**
## Description
when converting a hash value to string try to convert integer to string if value isn't a string

## Issues:
Refs: #682 

## How has this been tested:
 - [X] Covered by existing integration testing
 - [ ] Added integration testing to cover
 - [ ] Verified all A/C passes
     * [ ] develop
     * [ ] master
     * [ ] tag/other branch
 - [ ] Test environment
    * [ ] Shared Packet K8s cluster
    * [ ] New Packet K8s cluster
    * [ ] Kind cluster
 - [ ] Have not tested

## Types of changes:
 - [X] Bug fix (non-breaking change which fixes an issue)
 - [ ] New feature (non-breaking change which adds functionality)
 - [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
 - [ ] Documentation update

## Checklist:
**Documentation**
- [ ] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.
- [X] No updates required.

**Code Review**
- [ ] Does the test handle fatal exceptions, ie. rescue block

**Issue**
- [ ] Tasks in issue are checked off

Please review:
@taylor @wavell @lixuna

### CODE REVIEW

File: src/tasks/utils/utils.cr [hunk 1]
```diff
@@ -58,11 +58,11 @@ def loglevel
     end
   end
 
-  if ENV.has_key?("LOGLEVEL") 
+  if ENV.has_key?("LOGLEVEL")
```
Comment (williscool) [inline src/tasks/utils/utils.cr:61]: @sishbi can you remote the whitespace changes please?

File: src/tasks/utils/utils.cr [hunk 2]
```diff
@@ -291,6 +291,6 @@ def stdout_score(test_name)
 end
 
 def optional_key_as_string(totem_config, key_name)
-  "#{totem_config[key_name]? && totem_config[key_name].as_s?}"
+  "#{totem_config[key_name]? && (totem_config[key_name].as_s? || totem_config[key_name].as_i?)}"
```
Comment (williscool) [inline src/tasks/utils/utils.cr:294]: @sishbi can you add a comment with format of version number this fixes?

**Review by williscool — CHANGES_REQUESTED** [review]

File: src/tasks/utils/utils.cr (hunk 1 above)
Comment (sishbi) [reply src/tasks/utils/utils.cr:61]: As discussed in the meeting, this is a side-effect of my IDE fixing trailing whitespace issues.
I have modified the settings to only fix lines that are modified so hopefully that will resolve this going forward.

File: src/tasks/utils/utils.cr (hunk 2 above)
Comment (sishbi) [reply src/tasks/utils/utils.cr:296]: Comment added

**Pull request merged**